src/pages/registration/FilesAndLicensePanel.tsx

```diff
@@ -185,57 +178,50 @@ export const FilesAndLicensePanel = ({ uppy }: FilesAndLicensePanelProps) => {
                 )}
                 <Paper elevation={5} component={BackgroundDiv}>
                   <Typography variant="h2" sx={{ mb: '1rem' }}>
-                    {t('common.link')}
+                    {t('common.links')}
                   </Typography>
-                  {originalDoi ? (
-                    <DoiField canEditDoi={canEditFilesAndLinks} />
-                  ) : (
-                    <TextField
-                      fullWidth
-                      variant="filled"
-                      label={t('registration.files_and_license.link_to_resource')}
-                      disabled={!canEditFilesAndLinks}
-                      value={
-                        associatedLinkIndex >= 0 ? (associatedArtifacts[associatedLinkIndex] as AssociatedLink).id : ''
-                      }
-                      error={associatedLinkHasError}
-                      helperText={
-                        associatedLinkHasError
-                          ? (errors.associatedArtifacts?.[associatedLinkIndex] as FormikErrors<AssociatedLink>).id
-                          : null
-                      }
-                      data-testid={dataTestId.registrationWizard.files.linkToResourceField}
-                      onChange={(event) => {
-                        const inputValue = event.target.value;
-                        if (inputValue) {
-                          if (associatedLinkIndex < 0) {
-                            const newAssociatedLink: AssociatedLink = {
-                              type: 'AssociatedLink',
-                              id: inputValue,
-                            };
-                            push(newAssociatedLink);
-                            const nullAssociatedArtifactIndex = associatedArtifacts.findIndex(
-                              associatedArtifactIsNullArtifact
-                            );
-                            if (nullAssociatedArtifactIndex > -1) {
-                              remove(nullAssociatedArtifactIndex);
-                            }
-                          } else {
-                            const fieldName = `${name}[${associatedLinkIndex}].${SpecificLinkFieldNames.Id}`;
-                            setFieldValue(fieldName, inputValue);
-                            setFieldTouched(fieldName);
-                          }
-                        } else {
-                          const associatedArtifactsBeforeRemoval = associatedArtifacts.length;
-                          remove(associatedLinkIndex);
-                          if (associatedArtifactsBeforeRemoval === 1) {
-                            // Ensure field is set to touched even if it's empty
-                            setFieldTouched(name);
-                          }
+
+                  <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
+                    {values.doi && <LinkField fieldName="doi" label={t('common.doi')} />}
```

Comment:
`fieldName` her burde kanskje også vært fra en importert enum eller const? Litt usikker på hvor den lever best, så lenge dette er eneste plassen den brukes 🤔 
Det er et argument for å legge den inn i `FileFiledEnums` som skal liste opp alle feltnavn på filer-fanen og som ser slik ut nå
```
enum FileFieldNames {
  AssociatedArtifacts = 'associatedArtifacts',
}
```